src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -121,14 +121,18 @@ export class SchoolSelectField extends React.Component {
     if (formContext.reviewMode) {
       const {
         city,
+        country,
         name,
-        state
+        state,
+        street
       } = institutionSelected;
 
       return (
         <div>
-          <p>{name}</p>
-          <p>{`${city}, ${state}`}</p>
+          {name && <p>{name}</p>}
+          {street && <p>{street}</p>}
+          {(city || state) && <p>{`${city && city}${city && state && ', '}${state && state}`}</p>}
```

Comment:
![screen shot 2018-08-07 at 16 25 19](https://user-images.githubusercontent.com/4998130/43805832-83612630-9a5e-11e8-8bca-46b7e1aecd72.png)

Reply:
Not getting street address back from the service yet.